cleaners/windows_explorer.xml

```diff
@@ -50,14 +53,52 @@
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\Map Network Drive MRU"/>
     <!--  printer ports -->
     <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\PrnPortsMRU"/>
+
+
+    <!--  more MRUs -->
+
+    <!-- Completely deleted by CCleaner !!!
+    <action command="winreg" path="HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ComDlg32"/>
+    -->
```

Comment:
Please elaborate on this commented out action and your comment

Reply:
Was just a note as a kind of documentation... This line was for my opinion to hard, so I don't included it... Maybe a later editor of the cleaner will include this line... So he have "some docs" from me to it...